test/test_jit.py

```diff
@@ -3419,6 +3419,29 @@ def forward(self, x):
             .check("aten::mul") \
             .run(m.inlined_graph)
 
+    def test_code_with_constants(self):
+        """
+        Check that the `code_with_constants` property correctly returns graph CONSTANTS in the
+        CONSTANTS.cN format used in the output of the `code` property.
+        """
+        @torch.jit.script
+        def foo(x=torch.ones(1)):
+            return x
+
+        class Moddy(torch.nn.Module):
+            def __init__(self):
+                super(Moddy, self).__init__()
+
+            def forward(self, x):
+                return foo()
+
+        m = torch.jit.script(Moddy())
+        src, CONSTANTS = m.code_with_constants
+
+        self.assertEqual(CONSTANTS.c0, torch.ones(1))
+        self.assertEqual(src, m.code)
+
+
```

Comment:
As an extra bonus, it would be good to add a test that confirms that we can `save()` a model, then `load()` it, then call `code_with_constants` on it, and things will work as we expect. `self.getExportImportCopy` will be useful for that.

Reply:
Added